cmake fails on systems without lsb_release

```
CMake Error at CMakeLists.txt:234 (string):
  string sub-command STRIP requires two arguments.


CMake Error at CMakeLists.txt:235 (if):
  if given arguments:

    "STREQUAL" "vivid"

  Unknown arguments specified
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -231,7 +231,9 @@ set(UNITY_SCOPES_MAJOR_MINOR ${UNITY_SCOPES_MAJOR}.${UNITY_SCOPES_MINOR})
 set(UNITY_SCOPES_FULL_VERSION ${UNITY_SCOPES_MAJOR_MINOR}.${UNITY_SCOPES_MICRO})
 
 execute_process(COMMAND lsb_release -c -s OUTPUT_VARIABLE SERIES)
-string(STRIP ${SERIES} SERIES)
+if(NOT SERIES STREQUAL "")
+    string(STRIP ${SERIES} SERIES)
+endif()
 if (SERIES STREQUAL "vivid")
     # For backward compatibility. On Vivid, the soname started counting at 3: libunity-scopes.so.3.
     math(EXPR UNITY_SCOPES_SOVERSION "${UNITY_SCOPES_MINOR} + 3")

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -232,7 +232,7 @@ set(UNITY_SCOPES_FULL_VERSION ${UNITY_SCOPES_MAJOR_MINOR}.${UNITY_SCOPES_MICRO})
 
 execute_process(COMMAND lsb_release -c -s OUTPUT_VARIABLE SERIES)
 string(STRIP ${SERIES} SERIES)
-if (${SERIES} STREQUAL "vivid")
+if (SERIES STREQUAL "vivid")
     # For backward compatibility. On Vivid, the soname started counting at 3: libunity-scopes.so.3.
     math(EXPR UNITY_SCOPES_SOVERSION "${UNITY_SCOPES_MINOR} + 3")
 else()

